notebooks/01_prepare_data/wikidata.ipynb

```diff
@@ -0,0 +1,369 @@
+       "      <th>1</th>\n",
+       "      <td>Q47703</td>\n",
+       "      <td>Q130232</td>\n",
+       "      <td>drama film</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>Q47703</td>\n",
+       "      <td>Q52162262</td>\n",
+       "      <td>film based on literature</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>Q47703</td>\n",
+       "      <td>Q7444356</td>\n",
+       "      <td>gangster film</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>Q47703</td>\n",
+       "      <td>Q959790</td>\n",
+       "      <td>crime film</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>5</th>\n",
+       "      <td>Q47703</td>\n",
+       "      <td>Q243556</td>\n",
+       "      <td>The Godfather</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>6</th>\n",
+       "      <td>Q47703</td>\n",
+       "      <td>Q102642</td>\n",
+       "      <td>Diane Keaton</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7</th>\n",
+       "      <td>Q47703</td>\n",
+       "      <td>Q108622</td>\n",
+       "      <td>Talia Shire</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>Q47703</td>\n",
+       "      <td>Q1147417</td>\n",
+       "      <td>Simonetta Stefanelli</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>Q47703</td>\n",
+       "      <td>Q1352034</td>\n",
+       "      <td>Lenny Montana</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  original_entity linked_entities      name_linked_entities\n",
+       "1          Q47703         Q130232                drama film\n",
+       "2          Q47703       Q52162262  film based on literature\n",
+       "3          Q47703        Q7444356             gangster film\n",
+       "4          Q47703         Q959790                crime film\n",
+       "5          Q47703         Q243556             The Godfather\n",
+       "6          Q47703         Q102642              Diane Keaton\n",
+       "7          Q47703         Q108622               Talia Shire\n",
+       "8          Q47703        Q1147417      Simonetta Stefanelli\n",
+       "9          Q47703        Q1352034             Lenny Montana"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "result_linked[1:10]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualize KG using networkx"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import networkx as nx\n",
+    "import matplotlib.pyplot as plt\n",
```

Comment:
The graph is so nice. Do we want to show the names instead of the entity ids?

Reply:
changed this to show the names